README.md

```diff
@@ -13,7 +13,7 @@ Here is a **curated selection** of the metrics, tools and statistical tests incl
 
 |                       	| **Description** 	| **Selection of Included Functions** 	|
 |-----------------------	|-----------------	|--------------	|
-| **[Continuous](https://scores.readthedocs.io/en/latest/included.html#continuous)**        	|Scores for evaluating single-valued continuous forecasts.                  	|Mean Absolute Error (MAE), Mean Squared Error (MSE), Root Mean Squared Error (RMSE), Additive Bias, Multiplicative Bias, Pearson's Correlation Coefficient, Flip-Flop Index, Quantile loss, Murphy score.              	|
+| **[Continuous](https://scores.readthedocs.io/en/latest/included.html#continuous)**        	|Scores for evaluating single-valued continuous forecasts.                  	|Mean Absolute Error (MAE), Mean Squared Error (MSE), Root Mean Squared Error (RMSE), Additive Bias, Multiplicative Bias, Pearson's Correlation Coefficient, Flip-Flop Index, Quantile loss, Murphy score, consistent scores for expectiles and quantiles.              	|
```

Comment:
I think the term 'consistent' is going to be confusing for people, as it potentially implies that the other scores are somehow inconsistent, and it's just not clear what it's going to imply. Perhaps "statistically consistent" with a reference?

Reply:
I mean, the majority of metrics in `scores` aren't consistent scoring functions for expectiles and quantiles. I'll have to think further about this. I even considered no adding these scores to the readme table, as it will be a very niche group of users who use this capability. I think that more people will use the threshold weighted scores functions when we add it which will call the consistent scores. Since we don't mention all scores on this table, I am okay with not mentioning it at all. This capability will however be very novel to scores.

Reply:
I think it may be worth including them, it's just a matter of keeping it clear. Once I have read through the whole PR I'll see if I have any suggestions.